plugins/plugin.xml

```diff
@@ -26,14 +26,26 @@
         <config-file parent="/*" target="res/xml/config.xml">
             <feature name="OptimoveSDKPlugin">
                 <param name="android-package" value="com.optimove.android.cordova.OptimoveSDKPlugin" />
-                <param name="onload" value="true" />
             </feature>
-            
+            <preference name="AndroidXEnabled" value="true" />
         </config-file>
         <config-file parent="/manifest/application" target="AndroidManifest.xml">
-               
-            <provider android:authorities="${applicationId}.optimoveinitprovider" android:name="com.optimove.android.cordova.OptimoveInitProvider" android:exported="false" />
-            
+                     <!-- Optimove FCM handler -->
+            <service android:name="com.optimove.android.optimobile.FirebaseMessagingService" android:exported="false">
+                <intent-filter>
+                    <action android:name="com.google.firebase.MESSAGING_EVENT" />
+                </intent-filter>
+            </service>
+            <!-- Optimove push receiver  -->
+            <receiver android:name="com.optimove.android.cordova.PushReceiver" android:exported="false">
+                <intent-filter>
+                    <action android:name="com.optimove.push.RECEIVED" />
+                    <action android:name="com.optimove.push.OPENED" />
+                    <action android:name="com.optimove.push.BUTTON_CLICKED" />
```

Comment:
There might be a missing type for dismissed, perhaps compare to base SDK now, not just Cordova SDK.

There were some changes that were merged into the original Kumulos Android SDK but not all x-platform ones were update.

Reply:
I added dismissed action :)